### ISSUE TITLE
Auto-group songs in favourites playlist

### ISSUE THREAD

**Issue by jydv402** [vfsfitvnm/ViMusic#780]
### Feature description

Hello,
It would be good if there was a feature which auto-groups songs in the favourites based on the song language or genre.

### Why do you want this feature?

It would make easier to play songs of a single language. For example, when on a travel with family, it would be easier to play songs that they would also like to hear from our favourites playlist.

**Comment by vfsfitvnm** [comment]
I don't think this is possible - songs do not have information such as the language or the genre.
